src/modules/home/components/header/search-bar.tsx

```diff
@@ -36,13 +37,13 @@ const commonTypographyStyles = {
 
 interface SearchBarProps {
   activeNav: 'stays' | 'experiences';
+  setSelectedLocation: (newState: string) => void;
```

Comment:
I think you should also name this prop differently because you are not searching by location.